package.json

```diff
@@ -52,13 +52,15 @@
     "popper.js": "1.16.1",
     "prop-types": "15.8.1",
     "react": "16.14.0",
+    "react-bootstrap-typeahead": "^6.2.3",
     "react-dom": "16.14.0",
     "react-redux": "7.2.9",
     "react-router": "5.3.4",
     "react-router-dom": "5.3.4",
     "redux": "4.2.0",
     "regenerator-runtime": "0.13.11",
-    "tinymce": "5.10.7"
+    "tinymce": "5.10.7",
+    "uuid": "^9.0.0"
```

Comment:
what's this for?

Reply:
It was for an unique id, I've change it for date time